core/package/index.js

```diff
@@ -53,6 +53,9 @@ class Package {
       rootPath: {
         value: rootPath,
       },
+      symLinkDir: {
+        value: pkg.symLinkDir,
```

Comment:
Can you point me to an example of _any_ other usage in the ecosystem? I don't want to add arbitrary fields to package.json just to satisfy a vanishingly small percentage of use cases.